docs/CommandLineInterface.md

```diff
@@ -0,0 +1,64 @@
+# Command line interface
+
+This library provides experimental command line interface `ddcirb` to get the percentage of the tests
+that will be skipped for the current test run (only when using RSpec).
+
+## Usage
+
+This tool must be used on the same runner as your tests are running on with the same ENV variables.
+Run the command in the same folder where you usually run your tests. Gem datadog-ci must be installed.
+
+Available commands:
+
+- `bundle exec ddcirb skipped-tests` - outputs the percentage of skipped tests to stdout. Note that it runs your specs
+in dry run mode, don't forget to set RAILS_ENV=test environment variable.
+- `bundle exec ddcirb skipped-tests-estimate` - estimates the percentage of skipped tests and outputs to stdout without loading
+your test suite and running it in dry run mode. ATTENTION: this is considerably faster but could be very inaccurate.
+
+Example usage:
+
+```bash
+$ RAILS_ENV=test bundle exec ddcirb skipped-tests
```

Comment:
should we show an example of the output? Or is that somewhere else?

Reply:
the output just one line below

Reply:
0.45 is the output